source/material_model/visco_plastic.cc

```diff
@@ -67,6 +67,33 @@ namespace aspect
       return plastic_yielding;
     }
 
+
+
+    template <int dim>
+    bool
+    ViscoPlastic<dim>::
+    is_yielding(const MaterialModelInputs<dim> &in) const
+    {
+      Assert(in.n_evaluation_points() == 1, ExcInternalError());
+
+      /* The following returns whether or not the material is plastically yielding
+       * as documented in evaluate.
+       */
+      bool plastic_yielding = false;
```

Comment:
why don't you move the declaration down to the assignment? (I know that this is copy&paste).